contracts/rateproducer/README.md

```diff
@@ -1,28 +1,75 @@
+## Compilation
+This contract is build with eden `clsdk` compiler, to have the local environment configurated to compile this contract is needed to [take a look at this link](https://github.com/eoscommunity/demo-clsdk) to have it clear or just follow next steps to go forward.
+
+## Get Started
+### Ubuntu 20.04
+Run following commands to set local environment.
+```
+sudo apt-get update
+sudo apt-get install -yq    \
+    binaryen                \
+    build-essential         \
+    cmake                   \
+    gdb                     \
+    git                     \
+    libboost-all-dev        \
+    libcurl4-openssl-dev    \
+    libgmp-dev              \
+    libssl-dev              \
+    libusb-1.0-0-dev        \
+    pkg-config              \
+    wget
+
+
+mkdir ~/work
+
+cd ~/work
+wget https://github.com/WebAssembly/wasi-sdk/releases/download/wasi-sdk-12/wasi-sdk-12.0-linux.tar.gz
+tar xf wasi-sdk-12.0-linux.tar.gz
+
+cd ~/work
+wget https://github.com/eoscommunity/Eden/releases/download/sdk-v0.1.0-alpha/clsdk-ubuntu-20-04.tar.gz
+tar xf clsdk-ubuntu-20-04.tar.gz
+```
+
+Make sure to have next variables to compile the `rateproducer` contract:
+```
+export WASI_SDK_PREFIX=~/work/wasi-sdk-12.0
+export CLSDK_PREFIX=~/work/clsdk
+
+export PATH=$CLSDK_PREFIX/bin:$PATH
 ```
-Compile contract:
-eosio-cpp -R ricardian rateproducer.cpp -o rateproducer.wasm -abigen 
 
+Once before steps are done:
+```
+mkdir build
+cd build
+cmake `clsdk-cmake-args` ..
+make -j
+```
+
+```
 Add resources:
-cleos -u http://jungle2.cryptolions.io:80 system buyram rateproducer rateproducer "100 EOS"
+cleos -u https://jungle3.cryptolions.io system buyram rateproducer rateproducer "100 EOS"
 
 Publish Contract:
-cleos -u http://jungle2.cryptolions.io:80 set contract rateproducer ./ -p rateproducer@active
+cleos -u https://jungle3.cryptolions.io set contract rateproducer ./ -p rateproducer@active
```

Comment:
```bash
cleos -u https://jungle3.cryptolions.io set contract rateproducer ./ -p rateproducer@active
```

Reply:
@xavier506 what do you mean here?

Reply:
use code syntax highlighting 

https://docs.github.com/en/github/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks

Reply:
ahh ok ok thanks.